OpenMarket/OpenMarket/Product/Currency.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Currency.swift
+//  OpenMarket
+//
+//  Created by Jpush, Aaron on 2022/12/02.
+//
+
+import Foundation
+
+//enum Currency: String, Codable {
+//    case KRW = "KRW"
+//    case USD = "USD"
+//}
```

Comment:
Currenty 열거체를 따로 Product에서 분리하고 싶었는데 왜인지 모를 Xcode 오류로 인식하지 못해서.. 
의도 대로만 작성 하고 주석 처리 해둔 후 Product에 열거체를 남겨두었습니다

Reply:
제 Xcode에서는 잘 동작하는데, 혹시 어떠한 오류가 나오는지 코멘트 남겨주실수 있나요?

Reply:
```suggestion
enum Currency: String, Codable {
    case KRW
    case USD
}
```
로 사용해도, String 프로토콜을 따르기 때문에 rawValue가 필요없지 않을까요? 다른 의도가 있으신걸까요?

Reply:
cannot find Currency 오류가 떴었습니다 
제 Xcode만 가끔 오류를 뿜는데 되다 안되다 왔다갔다 하는 것 같더라구요 😭

Currency는 최초에 케이스 명을 소문자로 하려다 대문자로 바꾸면서 변경되지 않았던 부분이었습니다! 
수정하겠습니다!